openshift/templates/compliance-backend-bc.yaml

```diff
@@ -0,0 +1,90 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: builder-insights-compliance-backend
+
+parameters:
+- name: NAMESPACE
```

Comment:
2 keys in the same level, no indentation, is this correct?

Reply:
There are several instances of this in this 'parameters'  key

Reply:
it's required by `ocdeployer` https://github.com/RedHatInsights/e2e-deploy#how-do-template-variables-work.:
> By default, the 'NAMESPACE' variable is passed to all templates, this corresponds to the project name.
It just needs to be defined in parameters.

Reply:
The indentation is correct. The template is deployed successfully.